iceberg/iceberg-handler/src/test/queries/positive/show_create_iceberg_table.q

```diff
@@ -0,0 +1,7 @@
+DROP TABLE IF EXISTS ice_t;
+CREATE EXTERNAL TABLE ice_t (i int, s string, ts timestamp, d date) STORED BY ICEBERG;
+SHOW CREATE TABLE ice_t;
```

Comment:
Do we know what happens when we use the "old" partition syntax for identity transforms?
`CREATE TABLE tbl (a int, b int) STORED BY ICEBERG PARTITIONED BY (c string)`

Reply:
In this case, the partition spec will be among the table properties. I've added a new test case to cover this as well.

Reply:
sorry, I probably wasn't clear, I meant to ask about the `PARTITIONED BY (c string)` syntax (which creates a single, identity transform partition). But also testing the json property is a good call too.

Reply:
Great!

Reply:
Although...on second thought that won't produce a syntactically correct statement right? because we'd need to `c string` to be in the column definitions then like this:
```
CREATE TABLE tbl (a int, b int, c string) PARTITIONED BY SPEC(c) STORED BY ICEBERG
```

Reply:
My example was incorrect :), but the actual output will contain the additional column.

Reply:
Great :)